src/interfaces.ts

```diff
@@ -69,6 +69,7 @@ export interface CollectionActions<T> {
 
 interface UseCollectionResultBase<T> {
   items: ReadonlyArray<T>;
+  unpaginatedItems: ReadonlyArray<T>;
```

Comment:
I think perhaps `allFilteredItems` or `allMatchingItems` might be a more intuitive name, it took me a while to figure out what `unpaginatedItems` meant

Reply:
We already have `items` in the result that are sorted, filtered and paginated. The `unpaginatedItems` means: "same as items but not paginated".

How about: "allPageItems"?

Reply:
works for me